server/webhook_parser.go

```diff
@@ -255,9 +255,9 @@ func appendCommentNotifications(wh *webhook, verb string) {
 	jwh := wh.JiraWebhook
 	commentAuthor := mdUser(&jwh.Comment.UpdateAuthor)
 
-	message := fmt.Sprintf("%s %s %s:\n>%s",
-		commentAuthor, verb, jwh.mdKeySummaryLink(), jwh.Comment.Body)
-
+	message := fmt.Sprintf("%s %s %s:\n%s",
+		commentAuthor, verb, jwh.mdKeySummaryLink(), wh.text)
```

Comment:
@jfrerich I'd like to get your opinion on this. Before this PR, we were truncating (line 245) the text of the comment body before posting in channels, and before parsing to find mentions. But we used the non-truncated raw comment body for the content of the notification DMs. I think this inconsistency was by accident.

With the changes in this PR, the notification DMs are now also truncated because it is using the already modified `wh.text` instead of accessing `jwh.Comment.Body` directly. What are your thoughts on this change?

Reply:
In order to not change UX of the content in a DM notification, I actually think we change this to the following:

```suggestion
		commentAuthor, verb, jwh.mdKeySummaryLink(), quoteIssueComment(jwh.Comment.Body))
```

The existing truncate inconsistency can be discussed in another ticket

Reply:
@aidapira I'd like to try and enforce my suggestion above. Instead of using `wh.text` here, we can call `quoteIssueComment` like so, for two reasons:

- Preserve the behavior of sending the whole comment body to a user's DM notification
- Not rely on the order of operations before this function is called. For example, we implicitly rely on `wh.text` being modified before this function is called. If we apply my above suggestion, then it is not an issue.